rollingpin/frontends.py

```diff
@@ -127,31 +135,33 @@ def on_deploy_begin(self):
         self.start_time = time.time()
         print colorize("*** starting deploy", Color.BOLD(Color.GREEN))
 
-    def count_hosts(self):
-        return len(self.host_results)
-
-    def count_completed_hosts(self):
-        return sum(1 for v in self.host_results.itervalues() if v)
+    @inlineCallbacks
+    def on_enqueue(self, host, deferred):
+        if False:
```

Comment:
Debugging left in?

Reply:
nah, just a hack. added a comment to explain why. lemme know if it's still confusing.

Reply:
Oh neat!